subsys/mpsl/fem/Kconfig

```diff
@@ -47,6 +52,15 @@ config MPSL_FEM_NRF21540_GPIO
 	help
 	  FEM device is nRF21540 and its control mode is GPIO.
 
+config MPSL_FEM_NRF21540_GPIO_SPI
+	depends on MPSL_FEM_NRF21540_GPIO_SPI_SUPPORT
+	select NRFX_GPIOTE
+	select NRFX_PPI if SOC_SERIES_NRF52X
+	select NRFX_DPPI if SOC_SERIES_NRF53X
+	bool "nRF21540 front-end module in SPI mode"
```

Comment:
`GPIO/SPI mode` or `mixed GPIO and SPI mode` or similar

Reply:
Fixed